src/python/impactx/dashboard/Optimize/optimizeMain.py

```diff
@@ -0,0 +1,54 @@
+from trame.app import get_server
+from trame.widgets import vuetify
+
+# -----------------------------------------------------------------------------
+# Trame setup
+# -----------------------------------------------------------------------------
+
+server = get_server(client_type="vue2")
+state, ctrl = server.state, server.controller
+
+# -----------------------------------------------------------------------------
+# Helpful
+# -----------------------------------------------------------------------------
+
+
+class Optimize:

```

Comment:
Can you please add doc strings to the class?
```suggestion
class Optimize:
    """..."""
```